### PR TITLE
Skipping ACL creation on any change for select Account Ids.

### PR DESCRIPTION
**Pull request by Jimit09Gandhi**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/39449)

### CODE REVIEW

File: access-control/libs/aggregator/src/main/java/io/harness/aggregator/consumers/UserGroupChangeConsumerImpl.java [hunk 1]
```diff
@@ -83,6 +84,12 @@ public void consumeUpdateEvent(String id, UserGroupDBO updatedUserGroup) {
     if (!userGroup.isPresent()) {
       return;
     }
+    List<String> list = Arrays.asList("/ACCOUNT/2gf_S_jTQMuh2aiLSELxTw", "/ACCOUNT/px7xd_BFRCi-pfWPYXVjvw");
+
+    if (updatedUserGroup != null &&  list.stream().anyMatch(item ->  updatedUserGroup.getScopeIdentifier().startsWith(item))) {
+      log.info(String.format("RoleAssignmentChangeConsumerImpl: Skipping ACL creation for %s", updatedUserGroup.getScopeIdentifier()));
```
Comment (gn-harness) [inline access-control/libs/aggregator/src/main/java/io/harness/aggregator/consumers/UserGroupChangeConsumerImpl.java:90]: UserGroupChangeConsumerImpl*

**Review by gn-harness — APPROVED** [review]

**Review by ankit-harness — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39449)* status: 5 file(s) to review. 1 change requests. 

___
*[`access-control/libs/aggregator/src/main/java/io/harness/aggregator/consumers/RoleAssignmentChangeConsumerImpl.java` line 80 at r1](https://reviewable.io/reviews/harness/harness-core/39449#-NFjWubJ1akX4GAPpv5W:-NFjWubK8oKYyYX4zmMU:b40a65e) ([raw file](https://github.com/harness/harness-core/blob/c9909c92d3139a5a9268b73a59412cf8d57ec3de/access-control/libs/aggregator/src/main/java/io/harness/aggregator/consumers/RoleAssignmentChangeConsumerImpl.java#L80)):*
> ```Java
>   public void consumeCreateEvent(String id, RoleAssignmentDBO newRoleAssignmentDBO) {
>     List<String> list = Arrays.asList("/ACCOUNT/2gf_S_jTQMuh2aiLSELxTw", "/ACCOUNT/px7xd_BFRCi-pfWPYXVjvw");
>     if (newRoleAssignmentDBO != null && list.stream().anyMatch(item -> newRoleAssignmentDBO.getScopeIdentifier().startsWith(item))) {
> ```

Add a return statement

**Review by ankit-harness — APPROVED** [review]
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39449)* status: 4 file(s) to review.

**Review by ankit-harness — COMMENTED** [review]
:lgtm:

> Reviewed 4 of 4 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/39449#-:-NFjjFMGDxhqD37AGx8v:bnfp4nl)* status: :shipit: complete! ✓

**Comment by Jimit09Gandhi** [comment]
trigger compile

**Pull request merged**